### PR TITLE
Small pinnacle fixes

### PR DESCRIPTION
**Pull request by crcrewso**
This has a number of small pinnacle export fixes including better handling of some previously unused color constants. 

New Feature - flag --mrn appends the medical record number to the end of the name of the export folder

### CODE REVIEW

**Review by SimonBiggs — APPROVED** [review]
LGTM :)

I'll let @pchlap approve and merge though given this is his area of the codebase.

File: lib/pymedphys/cli/experimental/pinnacle.py
Comment (pchlap) [inline lib/pymedphys/cli/experimental/pinnacle.py:62]:
```suggestion
        help=("Ammend the Patient ID/MRN to the end of the name of the export folder"),
```

HRN should be MRN?

Comment (crcrewso) [reply lib/pymedphys/cli/experimental/pinnacle.py:62]: Thank you. Missed that one!

File: lib/pymedphys/cli/experimental/pinnacle.py
Comment (sjswerdloff) [inline lib/pymedphys/cli/experimental/pinnacle.py:62]: Amend

Comment (crcrewso) [reply lib/pymedphys/cli/experimental/pinnacle.py]: Thank you. Replaced with append as it is adding to the end of a string

**Comment by sjswerdloff** [comment]
I'm not sure why the Library/ Tests (Ubuntu, propagate) failed to find the cc-small-fixes references, but the check isn't listed as "Required".   I'm also not sure why a Notebook Review was generated (none of the changes could be reviewed in "Review NB".

**Comment by SimonBiggs** [comment]
> I'm not sure why the Library/ Tests (Ubuntu, propagate) failed to find the cc-small-fixes references

Unfortunately the propagate test requires pull/push access to the branch. Given @crcrewso is using a branch outside of the pymedphys org this one can't work.

**Comment by crcrewso** [comment]
Is there any quick documentation on how I should add external permissions to this branch. If not I can do my research tomorrow. I was wondering why it kept failing this.

**Comment by SimonBiggs** [comment]
> Is there any quick documentation on how I should add external permissions to this branch.

Given you already have permissions on the pymedphys repo, you can take your current clone and change the origin to be `git@github.com:pymedphys/pymedphys.git` instead of `git@github.com:crcrewso/pymedphys.git`:

https://docs.github.com/en/get-started/getting-started-with-git/managing-remote-repositories

Then just push your local branch up to the new origin, and make the pull request from that branch :).

**Comment by crcrewso** [comment]
@SimonBiggs I hope I've fixed this up as requested. I didn't even thing I should be using a branch on the main repository instead of my fork. I'll know for next time!

**Pull request closed without merging**
